plugins/auth/login.js

```diff
@@ -12,41 +12,39 @@ const uuid = require('uuid/v4');
  * @return Array                                    List of new routes
  */
 function addGuestRoute(server, config) {
-    return [{
-        method: ['GET'],
-        path: '/auth/login/guest/{web?}',
-        config: {
-            description: 'Login as an guest user',
-            notes: 'Authenticate an guest user',
-            tags: ['api', 'auth', 'login'],
-            auth: null,
-            handler: (request, reply) => {
-                // Check if guest is allowed to login
-                if (!config.allowGuestAccess) {
-                    return reply(boom.forbidden('Guest users are not allowed access'));
-                }
-
-                const username = `guest/${uuid()}`;
-                const profile = request.server.plugins.auth.generateProfile(
-                    username, null, ['user', 'guest'], {}
-                );
-
-                // Log that the user has authenticated
-                request.log(['auth'], `${username} has logged in`);
-
-                profile.token = request.server.plugins.auth.generateToken(
-                    profile, config.sessionTimeout
-                );
-                request.cookieAuth.set(profile);
-
-                if (request.params.web === 'web') {
-                    return reply('<script>window.close();</script>');
+    return [
+        {
```

Comment:
seems like an unnecessary newline
is there anything we can change in the rules to handle this issue?

Reply:
it's a prettier rule on items in an array being kept in individual lines. eslint has something similar but `prettier` overrides it to be off https://github.com/prettier/eslint-config-prettier/blob/master/index.js#L24
have to see if this can be tweaked. IMO this is better with multiple object items in array, not need to fiddle around beginning and ending of object paranthesis

Reply:
yea this is an intentional decision with prettier https://github.com/prettier/prettier/pull/6694#issuecomment-566898456 since we are adopting `prettier` we have to stuck with it's design philosophies 

https://prettier.io/docs/en/option-philosophy.html

Reply:
okay sounds good
as long as we are aware and okay with it